src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -119,11 +112,14 @@ app.config($routeProvider => {
     template: indexTemplate,
     reloadOnSearch: false,
     resolve: {
-      savedObjects: function(redirectWhenMissing, $route, kbnUrl, Promise, $rootScope, State) {
+      savedObjects: function(redirectWhenMissing, $route, kbnUrl, Promise, $rootScope) {
         const indexPatterns = getServices().indexPatterns;
         const savedSearchId = $route.current.params.id;
         return ensureDefaultIndexPattern(core, getServices().data, $rootScope, kbnUrl).then(() => {
```

Comment:
Could it be simplified?
 
```suggestion
        return ensureDefaultIndexPattern(core, data, $rootScope, kbnUrl).then(() => {
```

Reply:
of course, thx dear  🦅 - eye!  done